pkgs/applications/graphics/pdfcpu/default.nix

```
@@ -2,16 +2,36 @@
 
 buildGoModule rec {
   pname = "pdfcpu";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "pdfcpu";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-dEAlOKjNXL7zqlll6lqGmbopjdplDR3ewMMNu9TMsmw=";
+    hash = "sha256-410DIdsPXKMxnBL+1KX/DIPktqoB/ZxgF5GXEiqLcbw=";
+    leaveDotGit = true;
+    postFetch = ''
+      cd "$out"
+      git rev-parse HEAD > $out/COMMIT
+      date -u -d "@$(git log -1 --pretty=%ct)" "+%Y-%m-%dT%H:%M:%SZ" > $out/SOURCE_DATE_EPOCH
```

Why add the `_EPOCH` suffix?

```suggestion
      date -u -d "@$(git log -1 --pretty=%ct)" "+%Y-%m-%dT%H:%M:%SZ" > $out/SOURCE_DATE
```

You can even simplify this with:

```suggestion
git log -1 --pretty=%cd --date=format:'%Y-%m-%dT%H:%M:%SZ' > $out/SOURCE_DATE
```

---

I copied the code from another nix package, but I do not remember which.
`git log -1 --pretty=%ct` returns a unix timestamp, so that's probably why the file has the `_EPOCH` suffix.

---

I thought "epoch" means 1970-01-01 in computer science context...

---

I think that is the canonical definition, yet I've seen folks misuse epoch as seconds sind Jan 1st 1970, i.e. Unix time (and probably have done so myself 😅)